jax/_src/numpy/ufuncs.py

```diff
@@ -834,9 +834,29 @@ def log2(x: ArrayLike, /) -> Array:
   return lax.div(lax.log(x), lax.log(_constant_like(x, 2)))
 
 
-@implements(np.log10, module='numpy')
 @partial(jit, inline=True)
 def log10(x: ArrayLike, /) -> Array:
+  """Calculates the base-10 logarithm of x element-wise
+
+  LAX-backend implementation of :func:`numpy.log10`.
+
+  Args:
+    x: Input array
+
+  Returns:
+    An array containing the base-10 logarithm of each element in ``x``, promotes
+    to inexact dtype.
+
+  Examples:
+    >>> x1 = jnp.array([0.01, 0.1, 1, 10, 100, 1000])
+    >>> jnp.log10(x1)
+    Array([-1.9999999 , -0.99999994,  0.        ,  0.99999994,  1.9999999 ,
```

Comment:
Floating point outputs are problematic for doctests, because different backends may round results differently. One way to deal with this is to print the values within a `jnp.printoptions` context; you can check out some examples in `jax/_src/numpy.lax_numpy.py`